graphics.cpp

```diff
@@ -50,10 +50,12 @@ void Graphics::AddSprite(const Sprite& sprite, const SDL_Rect& destination,
 
 void Graphics::AddSprite(const Sprite& sprite, Location location, Perf* perf) {
   SDL_Rect source = sprite.GetSourceLocation();
```

Comment:
Make this "GetSourceAnimationFrame", and return the struct with the offset and rect